src/applications/combined-debt-portal/debt-letters/components/HowDoIPay.jsx

```diff
@@ -1,97 +1,109 @@
 import React from 'react';
 import PropTypes from 'prop-types';
+import { deductionCodes } from '../const/deduction-codes';
 
-const HowDoIPay = ({ userData }) => (
-  <>
-    <article className="vads-u-font-family--sans vads-u-padding-x--0">
-      <h2
-        id="howDoIPay"
-        className="vads-u-margin-top--4 vads-u-margin-bottom-2"
-      >
-        How to pay your VA debt
-      </h2>
+export const getDeductionDescription = code => {
+  const description = deductionCodes[code];
+  return <>{description || 'Unknown deduction code'}</>;
```

Comment:
@josephrlee 
Do we want to surface `Unknown deduction code`, I'm wondering if it would be better to return an empty string with just the deduction code.

Reply:
Yeah, that makes sense. I'll remove this bit.